contributing.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to AVA
+
+✨ Thanks for contributing to AVA! ✨
+
+## How can I contribute?
+
+### Improve documentation
+
+As a user of AVA you're the perfect candidate to help us improve our documentation. Typo corrections, error fixes, better explanations, more examples, etc. Open issues for things that could be improved. [Help translate our docs.](https://github.com/sindresorhus/ava-docs) Anything. Even improvements to this document.
+
+### Triage issues
```

Comment:
Is there a simpler word to replace "triage"? Had no idea what it meant until now.

Perhaps "improve issues" or something along those lines?